ONSdigital/rm-action-service
# Inherit fmt and checkstyle from parent

# Motivation and Context
Bump the parent pom version to include fmt and checkstyle configuration.
This removes the need of the custom fmt and checkstyle configuration

# What has changed
Remove `<executions>` blocks as they are now inherited from parent

# How to test?
1. Cause checkstyle issue i.e. force a line length of 100+ characters
1. Run `mvn install` and expect failure

# Links
https://trello.com/c/YUQ48iFw/172-automagically-format-code
